### PR TITLE
signup & login final check

### PR DESCRIPTION
**Pull request by NourAdawi**
- display error msg
- redirect after login

### CODE REVIEW

File: client/src/firebase/firebaseConfig.js [hunk 1]
```diff
@@ -14,3 +14,4 @@ const firebaseConfig = {
 // Initialize Firebase
 export const app = initializeApp(firebaseConfig);
 export const analytics = getAnalytics();
+setUserProperties(analytics, { favorite_food: 'apples' });
```
Comment (Amoodaa) [inline client/src/firebase/firebaseConfig.js:17]: whats this

File: client/src/components/Register/index.js [hunk 2]
```diff
@@ -18,24 +19,36 @@ export const Register = () => {
...
-      signUpWithEmail(email, password);
+    }
+    try {
+      await signUpWithEmail(email, password);
+      enqueueSnackbar('Your account has been created successfully !', {
+        variant: 'success',
+      });
+      history.push('/');
+    } catch (err) {
+      setAuthError(true);
```
Comment (Amoodaa) [inline client/src/components/Register/index.js:51]: you can use error coming from context

File: client/src/firebase/firebaseConfig.js (hunk 1 above)
Comment (NourAdawi) [reply client/src/firebase/firebaseConfig.js:17]: just for testing stuff and forgot to delete it

File: client/src/components/Register/index.js (hunk 2 above)
Comment (NourAdawi) [reply client/src/components/Register/index.js:51]: done

**Review by M7Mansour — APPROVED** [review]

**Pull request closed without merging**